diff -up gems/buildr-1.4.3/lib/buildr/java/packaging.rb.fixwarning gems/buildr-1.4.3/lib/buildr/java/packaging.rb
--- gems/buildr-1.4.3/lib/buildr/java/packaging.rb.fixwarning	2010-11-03 17:40:12.733084875 -0400
+++ gems/buildr-1.4.3/lib/buildr/java/packaging.rb	2010-11-03 17:40:19.220148599 -0400
@@ -59,7 +59,7 @@ module Buildr
             Zip::ZipInputStream::open(file.to_s) do |zip|
               while (entry = zip.get_next_entry)
                 if entry.name == 'META-INF/MANIFEST.MF'
-                  return Manifest.parse zip.read
+                  return Manifest.parse(zip.read)
                 end
               end
             end
